chart/kubeapps/templates/dashboard-config.yaml

```diff
@@ -17,6 +17,15 @@ data:
       gzip_static  on;
 
       location / {
+        # Support for ingress prefixes maintaining compatibility with the default /
+        # 1 - Exactly two fragment URLs for files existing inside of the public/ dir
+        # i.e /[prefix]/config.json => /config.json
```

Comment:
In kubectl proxy those two rewrites will be ignored as expected since they only match prefix defines in an ingress

In any case kubectl proxy does not work because of the header trim limitation.
 
![selection_795](https://user-images.githubusercontent.com/24523/50243802-c57ead00-0393-11e9-8289-da9f677d3432.png)

If you mean supporting multiple levels prefixes like /foo/kubeapps/ in the ingress, we do not support it and I think it's fine.

That scenario could be easily fixed if instead of providing all the files in `public` in the root i.e my-instance.com/favicon.png we add a prefix in a similar way we do with `static` `instance.com/public/favicon.png`

Having those two well defined directories would help with rewrites in the future.